### PR TITLE
Companies List() method

### PR DESCRIPTION
**Pull request by AkmalHanis**

### CODE REVIEW

File: src/main/java/CompanyList.java [hunk 1]
```diff
@@ -5,4 +5,16 @@
 public class CompanyList {
 
     ArrayList<Company> companies = new ArrayList<>();
+
+    public void list(){
+        if (companies.size() == 0){
```
Comment (kishore-a00) [inline src/main/java/CompanyList.java:10]: Perhaps include a space between the closing parenthesis and the opening curly brace.

File: src/main/java/CompanyList.java [hunk 2]
```diff
@@ -5,4 +5,16 @@
...
+            System.out.println("You do not have any companies in this list");
+        }
+        else {
```
Comment (kishore-a00) [inline src/main/java/CompanyList.java:13]: Perhaps move the else block up to line 12

**Review by kishore-a00 — COMMENTED** [review]
Looks good to merge, just a few minor fixes to make

File: src/main/java/CompanyList.java [hunk 3]
```diff
@@ -5,4 +5,16 @@
 public class CompanyList {
 
     ArrayList<Company> companies = new ArrayList<>();
+
+    public void list(){
```
Comment (kishore-a00) [inline src/main/java/CompanyList.java:9]: Perhaps consider renaming the method `list` to `listCompanyInformation` to differentiate between listing companies and listing venues

File: src/main/java/CompanyList.java [hunk 4]
```diff
@@ -5,4 +5,13 @@
 public class CompanyList {
 
     ArrayList<Company> companies = new ArrayList<>();
+
+    public void listCompanInyformation(){
+        if (companies.size() == 0) {
+            System.out.println("You do not have any companies in this list");}
```
Comment (Manoj364) [inline src/main/java/CompanyList.java:11]: Perhaps you can move the  `}` to the next line

File: src/main/java/CompanyList.java [hunk 5]
```diff
@@ -5,4 +5,13 @@
...
+        else {
+            for (int i = 0; i < companies.size(); i += 1) {
+                System.out.println((i+1) + ". " + companies.get(1));}
```
Comment (Manoj364) [inline src/main/java/CompanyList.java:14]: Perhaps you can move the  `}` to the next line here as well

**Comment by Manoj364** [comment]
Looks good to merge!

**Pull request closed without merging**